app/controllers/posts_controller.rb

```diff
@@ -19,17 +19,17 @@ def create
     post.user = current_user
     # post.user_id = @user.id
```

Comment:
- [ ] OPTIONAL
Your app does not solve the N+1 problem when fetching all posts and their comments for a user. An example of this in a `posts#index` actions should be similar to:
```ruby
@posts = @user.posts.includes(:comments) 
```

instead of 

```ruby
@posts = @user.posts

Reply:
Great! thank you.